docs/query-builder.txt

```diff
@@ -46,6 +46,14 @@ The following example shows the syntax of a query builder call:
    DB::collection('<collection name>')
        // chain methods by using the "->" object operator
        ->get();
+.. tip::
+
+   Before using the ``DB::collection()`` method, ensure that you specify MongoDB as your application's
+   default database connection. For instructions on setting the database connection,
+   see the :ref:`laravel-quick-start-connect-to-mongodb` step in the Quick Start.
+
+   If MongoDB is not your application's default database, you can use the ``DB::connection()`` method
+   to specify a MongoDB connection. Pass a value of ``"mongodb"`` to the ``connection()`` method.
```

Comment:
This should mention that one should pass the connection name (for which we use "mongodb" in our examples). Also, I suggest using a code snippet for this purpose:
```suggestion
   If MongoDB is not your application's default database, you can use the ``DB::connection()`` method
   to specify a MongoDB connection. Pass a the name of the connection to the ``connection()`` method:
   
   .. code-block:: php

      $connection = DB::connection('mongodb');
```